.github/workflows/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
```

Comment:
Only actions should BFS in the workflows directory. CODEOWNERS can be in one of a few locations including the .github dir (see the doc for details)

Reply:
oops yeah, thanks